cmd/schema.go

```diff
@@ -46,38 +46,66 @@ environment variables. The schema flags are:
 
 // SetFlags sets the flags.
 func (cmd *SchemaCmd) SetFlags(f *flag.FlagSet) {
+	f.StringVar(&cmd.source, "source", "", "Flag for specifying source DB, (accepted values are \"PostgreSQL\", \"MySQL\", and \"DynamoDB\")")
+	f.StringVar(&cmd.sourceProfile, "source-profile", "", "Flag for specifying connection profile for source database")
```

Comment:
Should we add an example of source-profile and target profile in the desc?

Reply:
I have added examples for both now. Done.